### PR TITLE
[SessionD] Save bundled session context into SessionConfig 

### PR DESCRIPTION
**Pull request by themarwhal**
## Summary
Recently, proto messages `CommonSessionContext` and `RatSpecificContext` were added to the `LocalSessionManager` server so that different types of contexts can be bundled together. 
This diff adds the two new messages to `SessionConfig` so that it can be saved in `SessionStore` properly. 

Also this change renames one of the WLAN specific fields from `hardware_addr` to `mac_addr_binary` because the latter is more descriptive. 

## Test Plan
- [x] SessionD unit tests 
- [x] CWF Integ Test 
- [x] LTE Integ Test 

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: feg/gateway/services/aaa/servicers/accounting.go [hunk 1]
```diff
@@ -333,7 +333,7 @@ func createSessionOnSessionManager(mac net.HardwareAddr, subscriberId *lte_proto
 		RatSpecificContext: &lte_protos.RatSpecificContext{
 			Context: &lte_protos.RatSpecificContext_WlanContext{
 				WlanContext: &lte_protos.WLANSessionContext{
-					HardwareAddr:    mac,
+					MacAddrBinary:   mac,
```
Comment (uri200) [inline feg/gateway/services/aaa/servicers/accounting.go:336]: just curios, why is the mac address a binary?

File: lte/gateway/c/session_manager/StoredState.cpp [hunk 2]
```diff
@@ -66,6 +66,14 @@ std::string serialize_stored_session_config(const SessionConfig& stored) {
   marshaled["bearer_id"]         = std::to_string(stored.bearer_id);
   marshaled["qos_info"]          = serialize_stored_qos_info(stored.qos_info);
 
+  std::string common_context;
+  stored.common_context.SerializeToString(&common_context);
```
Comment (uri200) [inline lte/gateway/c/session_manager/StoredState.cpp]: if this is a protobuf, check SerializeAsString (funny name :D) method. It may save you some

Comment (uri200) [reply lte/gateway/c/session_manager/StoredState.cpp]: For the counting algo pr I am doing I also did the same, but eventually I decided to use the regular hard coded way..... I am trying to remember why I decided to use the ugly way, but I think it was due to the default values protos gives when the field is missing.

Comment (themarwhal) [reply lte/gateway/c/session_manager/StoredState.cpp]: How is `SerializeAsString` different from `SerializeToString` (also protobuf serialization func) ?

File: feg/gateway/services/aaa/servicers/accounting.go (hunk 1 above)
Comment (themarwhal) [reply feg/gateway/services/aaa/servicers/accounting.go:336]: I'm not exactly sure on the details, but it looks like when it is passed up to SessionProxy, it expects a bytes value.

Comment (uri200) [reply feg/gateway/services/aaa/servicers/accounting.go:336]: Probably leak of AVP then

File: lte/gateway/c/session_manager/StoredState.cpp (hunk 2 above)
Comment (uri200) [reply lte/gateway/c/session_manager/StoredState.cpp]: It gives you the string. You don't need to pass a reference to a string, so you can use it as
`marshaled["common_context"] = stored.common_context.SerializeAsString(&common_context);`

Comment (themarwhal) [reply lte/gateway/c/session_manager/StoredState.cpp]: Ohh I see thanks! That shortens it a bit :D

**Review by uri200 — APPROVED** [review]
👍

**Review by mpgermano — APPROVED** [review]

**Pull request merged**